use-cases/eurac/trainer.py

```diff
@@ -92,7 +93,8 @@ def __init__(
         self.save_parameters(**self.locals2params(locals()))
 
     @suppress_workers_print
-    @profile_torch_trainer
+    # @profile_torch_trainer
```

Comment:
Make sure that decorators can be used together (for future PRs, though)

Reply:
I think that as soon as I fix the other issue with `init()` and `cleanup()`, this kinda solves itself 😎